dev-tools/export_dashboards.py

```diff
@@ -119,7 +115,7 @@ def main():
     args = parser.parse_args()
 
     if args.index is None:
-        args.index = args.beat.lower() + "-*"
+        args.index = args.beat.lower()
```

Comment:
Yes, the index pattern is stored under `/.kibana/index-pattern/metricbeat`, and not `/.kibana/index-pattern/metricbeat-*`. I suggest to remove exporting the index pattern from the `export_dashboards.py`, as we don't want to overwrite the generated one.